ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -228,19 +241,11 @@ struct ZipBuilderTool: ParsableCommand {
 
     // Populate the platforms list if it's empty. This isn't a great spot, but the argument parser
     // can't specify a default for arrays.
-    let platformsToBuild = !platforms.isEmpty ? platforms : TargetPlatform.allCases
+    let platformsToBuild = !platforms.isEmpty ? platforms : Platform.allCases
     let builder = ZipBuilder(paths: paths,
                              platforms: platformsToBuild,
                              dynamicFrameworks: dynamic,
                              customSpecRepos: customSpecRepos)
-    let projectDir = FileManager.default.temporaryDirectory(withName: "project")
-
-    // If it exists, remove it before we re-create it. This is simpler than removing all objects.
-    if FileManager.default.directoryExists(at: projectDir) {
-      try FileManager.default.removeItem(at: projectDir)
-    }
-
-    CocoaPodUtils.podInstallPrepare(inProjectDir: projectDir, paths: paths)
```

Comment:
What's the reason for removing this?

Reply:
There is now a separate projectDir for each platform.